test/e2e-tests/tests/pipelines_test.go

```diff
@@ -12,7 +12,7 @@ import (
 
 const (
 	AIEdgeE2EPipelineDirectoryRelativePath = "../../../pipelines/tekton/aiedge-e2e"
-	PipelineRunFileRelativePath            = AIEdgeE2EPipelineDirectoryRelativePath + "/aiedge-e2e.pipelinerun.yaml"
+	PipelineRunFileRelativePath            = AIEdgeE2EPipelineDirectoryRelativePath + "/aiedge-e2e.bike-rentals.pipelinerun.yaml"
```

Comment:
```suggestion
	PipelineRunFileRelativePath            = AIEdgeE2EPipelineDirectoryRelativePath + "/aiedge-e2e.tensorflow-housing.pipelinerun.yaml"
```

I wonder if this would be a better one to use, now that we have a pipelinerun example for it? We care more about this OpenVINO-based one, and IIRC it's much quicker to run. WDYT?

Reply:
Yeah no problem. I think it really could be both aswell

Reply:
Sure, if it's not too much effort, adding a second test might be good. If you do, consider documenting how to run individual tests, alongside how to run the entire suite.

Reply:
```suggestion
	PipelineRunFileRelativePath            = filepath.Join(AIEdgeE2EPipelineDirectoryRelativePath, "aiedge-e2e.bike-rentals.pipelinerun.yaml")
```

I think this will also make it less platform dependent. You'd need to add `"path/filepath"` to the imports.